en/simulation/gazebo_vehicles.md

```diff
@@ -27,6 +27,22 @@ make px4_sitl gazebo
 make px4_sitl gazebo_iris_opt_flow
 ```
 
+### Quadrotor with Depth Camera
+
+These models have a depth camera attached, modelled on the Intel® RealSense™ D455.
```

Comment:
Is it worth including some of the detail you put in the autopilot PR? Specifically the frame size, and FOV? I'm not saying it is - I am "wondering" if this would make any difference to consumers/users of the model.